frontend/static/js/charts/block-size.js

```diff
@@ -20,5 +20,5 @@ function preprocess(input) {
 }
 
 function chartDefinition(d, movingAverage) {
-  return lineChart(d, NAME, movingAverage, PRECISION, START_DATE, ANNOTATIONS);
+  return lineChart(d, NAME, movingAverage, PRECISION, START_DATE, ANNOTATIONS, (v) => formatWithSIPrefix(v, "B"));
```

Comment:
> * Added SI prefix formatting to block size chart (see images)

Could you explain the motivation for this change? Going from byte-precision to 100-KB precision seems excessive to me. Could this please be at least 10s of KB precision (i.e., `x.yz MB`)?